### PR TITLE
Parquet doc tweaks

### PR DESCRIPTION
**Pull request by tustvold**
# Which issue does this PR close?

Closes #.

# Rationale for this change
 

Makes the docs a little easier to navigate

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: parquet/src/arrow/async_reader/mod.rs [hunk 1]
```diff
@@ -263,6 +265,39 @@ impl<T: AsyncFileReader + Send + 'static> ParquetRecordBatchStreamBuilder<T> {
...
+    /// # writer.write(&batch).unwrap();
+    /// # writer.close().unwrap();
+    /// #
+    /// let mut file = tokio::fs::File::from_std(file);
+    /// let meta = ArrowReaderMetadata::load_async(&mut file, Default::default()).await.unwrap();
+    /// let mut a = ParquetRecordBatchStreamBuilder::new_with_metadata(
+    ///     file.try_clone().await.unwrap(),
+    ///     meta.clone()
+    /// ).build().unwrap();
+    /// let mut b = ParquetRecordBatchStreamBuilder::new_with_metadata(file, meta).build().unwrap();
```
Comment (alamb) [inline parquet/src/arrow/async_reader/mod.rs:295]: 👍

File: parquet/src/arrow/mod.rs [hunk 2]
```diff
@@ -25,47 +25,39 @@
...
 //! let batch = RecordBatch::try_from_iter(vec![
 //!   ("id", Arc::new(ids) as ArrayRef),
 //!   ("val", Arc::new(vals) as ArrayRef),
 //! ]).unwrap();
 //!
-//! let file = File::create("data.parquet").unwrap();
+//! let file = tempfile().unwrap();
 //!
-//! let mut writer = ArrowWriter::try_new(file, batch.schema(), None).unwrap();
+//! let props = WriterProperties::builder()
```
Comment (alamb) [inline parquet/src/arrow/mod.rs:44]:
```suggestion
//! // WriterProperties  can be used to set Parquet file options
//! let props = WriterProperties::builder()
```

Comment (alamb) [reply parquet/src/arrow/mod.rs:44]: Maybe it would help to draw the readers's attention to what this is doing

**Review by alamb — APPROVED** [review]
Looks like a nice improvement to me. Thank you @tustvold

File: parquet/src/arrow/mod.rs (hunk 2 above)
Comment (tustvold) [inline parquet/src/arrow/mod.rs:44]:
```suggestion
//! // WriterProperties can be used to set Parquet file options
//! let props = WriterProperties::builder()
```

**Pull request merged**